Components/XandYLabelsGraph/index.js

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import Paragraph from "../Paragraph";
+import InputTypeText from "../InputTypeText";
+import Heading from "../Heading";
+import { StyledInputTypeSubmit } from "../StyledInputTypeSubmit";
+import CenteredDiv from "../CenteredDiv";
+
+export default function XandYLabelsGraph({
+  onXLabelChange,
+  onYLabelChange,
+  hasEnteredYLabel,
+}) {
+  const xKey = "age";
+  const yKey = "boys-height";
+
+  function handleSubmit(event) {
+    event.preventDefault();
+  }
+
+  function handleAxesLabels() {
+    alert("Labels for the x-axis and the y-axis of the graph are assigned");
+  }
+
+  return (
+    <>
+      {xKey != "" && yKey != "" ? (
```

Comment:
this ternary doesn't really have a purpose at the moment, because you set the xKey and the yKey in line 13 and 14, so it can never be empty strings. I understand you do this for the future, but in general we only implement code when it is relavant to the feature, so only implement this ternary when the xKey and yKey will be actually variable :)

Reply:
Removed conditional rendering of xKey and yKey